client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -597,7 +622,8 @@ class DeltaSharingRestClient(
       )
       getCDFFilesByPage(target)
     } else {
-      getNDJson(target, requireVersion = false)
+      val response = getNDJson(target, requireVersion = false)
+      (response.version, response.respondedFormat, response.lines)
```

Comment:
```suggestion
      val response = getNDJson(target, requireVersion = false)
      val (filteredLines, _) = maybeExtractEndStreamAction(response.lines)
      (response.version, response.respondedFormat, filteredLines)
```

Reply:
cdf is not supported yet, let me make the change in the CDF PR later.